packages/react/src/index.tsx

```diff
@@ -76,6 +77,7 @@ export const FigspecFrameViewer = forwardRef<
       onNodeSelect,
       onPositionChange,
       onScaleChange,
+      link,
```

Comment:
Since no casing conversion required for this prop, it's better to keep it included in the `rest` IMO.

Reply:
I thought of that, but I figured that if `link` is passed via `rest`, it feels a bit obscure as it's not implicit. Nonetheless, having it via rest is pretty good as well! I updated the code, so in fact the only change is adding `link` as a type.